src/Connectors/test/Connector.EF6Core.Test/OracleDbTypeLocatorTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Steeltoe.Connector.Oracle;
+using Xunit;
+
+#if NET461
```

Comment:
Should we remove this directive for the 3.x line?

Reply:
@kenswan this test would fail... The latest Oracle package for EF6 still only works with .NET Framework. Alternatively we could skip or remove the test until such time as they do support Core or .Net 5, but leaving it this way is a reminder that it only works on Framework

Reply:
Ah, thanks for that clarification!